### PR TITLE
adjust tc to floating toolbar and dimmed style changes

### PR DESCRIPTION
**Pull request by jbinda**
I have update Group test cases to adjust after changed dimmed styling when navigate in nested structure

### CODE REVIEW

File: test-cases/gutenberg/group.md [hunk 1]
```diff
@@ -107,21 +107,18 @@ Total space from screen edge to content should equals `16px` in each state (incl
 
 ##### TC007
 
-### Nested selection cause applying dimmed style on the rest of blocks
+### Nested selection do not cause applying dimmed style on the rest of blocks
```
Comment (pinarol) [inline test-cases/gutenberg/group.md:110]: We can remove this test case, let's also remove it from the sanity-tests.md file. (No need to update the TC numbers for now)

File: test-cases/gutenberg/group.md [hunk 2]
```diff
@@ -52,12 +52,12 @@ Expected look:
 
 Expected look:  
 ![PlaceholderNestes](../resources/group-placeholder-nest-1.png)
- - Post title is not dimmed
+ - Post title **is not** dimmed
```
Comment (pinarol) [inline test-cases/gutenberg/group.md:55]: We can remove these lines, we can accept the not dimming as the default behavior anymore.

File: test-cases/gutenberg/group.md (hunk 1 above)
Comment (jbinda) [reply test-cases/gutenberg/group.md]: fixed

File: test-cases/gutenberg/group.md (hunk 2 above)
Comment (jbinda) [reply test-cases/gutenberg/group.md]: line removed

File: test-cases/gutenberg/group.md [hunk 3]
```diff
@@ -105,35 +103,16 @@ Total space from screen edge to content should equals `16px` in each state (incl
...
 
 ### Breadcrumbs on FloatingToolbar is properly displayed
 
 -   Add a `Group` block
 -   Create nested structure
--   Check if each time you change selection `Breadcrumbs` show focused block title and focused block icon
--   Check if each time you change selection `Breadcrumbs` show top most parent block icon
--   Check if `Breadcrumbs` renders navigation up button
+-   Expect only selected block title is visible in `Breadcrumbs` on root list level
+-   Expect focused block title and focused block icon is visible in `Breadcrumbs` on deeper levels
```
Comment (lukewalczak) [inline test-cases/gutenberg/group.md:113]:
```suggestion
-   Expect focused block title and focused block icon are visible in `Breadcrumbs` on deeper levels
```

**Pull request merged**